UITests/ReaderViewUITests.swift

```diff
@@ -0,0 +1,111 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import WebKit
+
+class ReaderViewUITests: KIFTestCase, UITextFieldDelegate {
+    private var webRoot: String!
+
+    override func setUp() {
+        webRoot = SimplePageServer.start()
+    }
+
+    /**
+    * Tests reader view UI with reader optimized content
+    */
+    func testReaderViewUI() {
+        loadReaderContent()
+        addToReadingList()
+        markAsReadFromReaderView()
+        markAsUnreadFromReaderView()
+        markAsReadFromReadingList()
+        markAsUnreadFromReadingList()
+        removeFromReadingList()
+        addToReadingList()
+        removeFromReadingListInView()
+    }
```

Comment:
Instead of having a single `testX` method that calls a bunch of helper functions, can each of these helper methods be a separate test? There are some advantages to having a modular breakdown of tests, like being able to trigger them individually in Xcode.